react/features/old-client-notification/middleware.tsx

```diff
@@ -1,16 +1,16 @@
-// @flow
-
 import React from 'react';
```

Comment:
Why does this middleware need to be tsx ?

Reply:
Because it has JSX code

Reply:
Oh, I had missed that old description component thing!